e2e/cypress/integration/channel/more_channels_spec.js

```diff
@@ -141,6 +141,42 @@ describe('Channels', () => {
         // * Assert that archived channel doesn't show up in LHS list
         cy.get('#publicChannelList').should('not.contain', testChannel.display_name);
     });
+
+    it('MM-T1702 Search works when changing public/archived options in the dropdown', () => {
```

Comment:
For ease of test maintenance, each test block should be able to run in isolation. Please try to run this with `it.only` and ensure that it still work as expected.